clang/lib/Sema/CheckExprLifetime.cpp

```diff
@@ -1420,9 +1446,18 @@ static void checkExprLifetimeImpl(Sema &SemaRef,
              ? IndirectLocalPathEntry::LifetimeBoundCall
              : IndirectLocalPathEntry::GslPointerAssignment,
          Init});
+  } else if (LK == LK_LifetimeCapture) {
```

Comment:
Consider replacing with a switch-case.

Reply:
Done.

Reply:
You might not be looking at the latest changes. This was switched to a switch :)